.gitignore

```diff
@@ -9,3 +9,4 @@ dist/
 .vscode
 .venv
 prof/
+*.cache
```

Comment:
If we're storing cache files somewhere, the "current directory" seems like the wrong place to do so. The [platformdirs](https://github.com/tox-dev/platformdirs) package has a `user_cache_dir` function to provide the platform-specific directory where cache files should be stored. I suggest adding as dependency and using it.

Reply:
Yeah, I wasn't quite sure where to put it; `pipx` is pretty new to me and I'm used to just creating a virtual environment with `python3 -m venv venv` so I wasn't sure where pipx would store any supplemental files. I'll check it out.